BLSQ/iaso
# IA-3024: Not displaying permissions related to modules which are not linked to the account

Explain what problem this PR is resolving
- Not displaying permissions related to modules which are not linked to the account
Related JIRA tickets : [IA-3024](https://bluesquare.atlassian.net/browse/IA-3024)

## Self proofreading checklist


## Changes
- Check if grouped permissions is empty and skip if it is

## How to test
- Remove some modules from the current account
- Check if some permissions groups still appearing

## Print screen / video
[Screencast from 2024-05-24 16-02-47.webm](https://github.com/BLSQ/iaso/assets/19631540/d81f613e-be07-48cb-8587-d8154bf0f007)


[IA-3024]: https://bluesquare.atlassian.net/browse/IA-3024?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
